channeldb/models/channel_edge_policy.go

```diff
@@ -60,6 +60,15 @@ type ChannelEdgePolicy struct {
 	// HTLCs for each millionth of a satoshi forwarded.
 	FeeProportionalMillionths lnwire.MilliSatoshi
 
+	// InboundFeeBaseMSat is the base HTLC fee that will be subtracted for
```

Comment:
Could you instead add `ExtraOpaqueData lnwire.ExtraOpaqueData` on this struct and set it in `addToTopologyChange`? That way we don't have to add redundant data to `ChannelUpdate` and `ChannelEdgeUpdate` because the inbound fees are in the opaque tlv blob. We can then use this to pass that blob through to `marshalTopologyChange` and extract inbound fees using `extractInboundFeeSafe`. We can then as well set `CustomRecords` on the proto message preparing it with `marshalExtraOpaqueData`, similar to the approach that was taken in `marshalDBRoutingPolicy`.